python/paddle/incubate/operators/unzip.py

```diff
@@ -12,23 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from paddle.base.data_feeder import check_variable_and_dtype
+from paddle.base.data_feeder import check_type, check_variable_and_dtype
 from paddle.base.layer_helper import LayerHelper
 
 
-def unzip(input, lod):
+def unzip(input, lod, len):
```

Comment:
The meaning of new parameter `len` should be explained in the Args below

Reply:
will fix in the next pr